building-with-ocean/asset-hosting.md

```diff
@@ -17,134 +17,136 @@ On Ocean Marketplace, a publisher must provide the link to the asset during publ
 
 Publishers can choose any hosting service of their choice. The below section explains how to use commonly used hosting services with Ocean Marketplace.
 
-#### Google Drive
```

Comment:
Please update the description of this document. Currently is: "Tutorial to publish assets using hosting services like Google Drive and Azure."